lsp-mode.el

```diff
@@ -3916,14 +3916,20 @@ WORKSPACE is the active workspace."
       ;; screwed up with a previous Content-Length
       (error "No Content-Length header"))))
 
+(defun lsp--trim-left (s)
```

Comment:
why not using `s-trim-left`, we depend on `s`?

Reply:
still haven't got cpptools to work.  we can launch the server, but when we run `lsp-describe-thing-at-point`, it shows the server stuck in "Initializing...".  Hoping we can get it working!

Reply:
I had a question about it a couple of weeks ago and when I searched in the cpptools repo the maintainers were basically saying that it will work only with vscode and to try ccls/cquery/clangd. If you need help you may ping me in it gitter.